pyanaconda/modules/subscription/installation.py

```diff
@@ -191,6 +192,22 @@ def _copy_file(self, file_path, target_file_path):
         shutil.copy(file_path, target_file_path)
         return True
 
+    def _copy_folder(self, folder_path, target_folder_path):
```

Comment:
Maybe this could go to `pyanaconda/core/path.py` as it looks like quite possibly useful elsewhere going forward. :)

Reply:
moved it!